src/azul/plugins/utilities.py

```diff
@@ -0,0 +1,22 @@
+from typing import Optional
+
+from azul import require
+
+
+def unify_string_pair(str_one: Optional[str], str_two: Optional[str]) -> Optional[str]:
+    """
+    Helper method to unify two strings into one by a blank space character.
```

Comment:
```suggestion
    Helper method to unify two strings into one separated by a blank space character.
```
Also this function could benefit from some doctests.

Reply:
I removed the main body of the docstrings since the doctest showcase expected behavior. Let me know if you disagree.